langchain_tools_demo/main.py

```diff
@@ -153,6 +165,20 @@ async def reset(request: Request):
     request.session.clear()
 
 
+def init_app(cfg: AppConfig) -> FastAPI:
+    # FastAPI setup
+    app = FastAPI(lifespan=lifespan)
+    app.state.client_id = cfg.clientId
+    # app.state.orchestration = cfg.orchestration
+    app.include_router(routes)
+    app.mount("/static", StaticFiles(directory="static"), name="static")
+    # TODO: set secret_key for production
+    app.add_middleware(SessionMiddleware, secret_key="SECRET_KEY")
+    return app
+
+
 if __name__ == "__main__":
     PORT = int(os.getenv("PORT", default=8081))
+    cfg = parse_config("./config.yml")
+    app = init_app(cfg)
     uvicorn.run(app, host="0.0.0.0", port=PORT)
```

Comment:
I think this is ignoring the port set in the app

Reply:
Updating this. I'm keeping this `if __name__` for now, but eventually we will remove it during the last step. Wanted to make sure that this changes does not cause any change to existing instructions yet.